SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Ingestion/IServiceCall.cs

```diff
@@ -1,21 +1,43 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.AppCenter.Ingestion.Models;
 
 namespace Microsoft.AppCenter.Ingestion
 {
     public interface IServiceCall : IDisposable
     {
```

Comment:
`ServiceCall` doesn't contain retry logic anymore;
`Task ExecuteAsync();` replaced by `ContinueWith` callback to avoid long-running tasks;
Contains state getters now;

Reply:
how is continuewith not impacted by the same problem?

Reply:
This is not a `System.Task`. Just callback and method with the similar name to make the code a bit easier to understand